extern/boostd-data/svc/svc_test.go

```diff
@@ -46,27 +46,26 @@ var testCouchSettings = couchbase.DBSettings{
 }
 
 func TestService(t *testing.T) {
-	logging.SetLogLevel("*", "debug")
+	_ = logging.SetLogLevel("*", "debug")
 
-	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
```

Comment:
Do these tests take 5 minutes?

Reply:
CouchBase tests are taking longer. Not 5 minutes though. Since, I am removing all CB tests from this PR, I will reset all contexts to 5 Seconds.